packages/headless/src/controllers/core/did-you-mean/headless-core-did-you-mean.ts

```diff
@@ -21,6 +22,12 @@ import {
 
 export type {QueryCorrection, WordCorrection};
 
+export interface DidYouMeanProps {
+  /**
+   * Whether to automatically correct queries that will return no results.
```

Comment:
We will need more meat for the documentation. 

It needs to explain what is an "automatically corrected query", when it happens, what's the impact on the UX, etc.

I don't think an implementer would be able to understand what this option will actually change in their page when flipped to true/false.

We also need to explain the default behaviour when the option is left undefined.

Reply:
Hmm I see what you mean. I'll try to find a clearer description...